lua-resty-websocket-kong-0.1.0-1.rockspec

```diff
@@ -0,0 +1,24 @@
+package = "lua-resty-websocket-kong"
+version = "0.1.0"
+source = {
+   url = "git://github.com/kong/lua-resty-websocket",
+}
+description = {
+   summary = "Kong-managed fork of lua-resty-websocket",
+   detailed = [[
+      lua-resty-websocket-kong is a fork of the OpenResty websocket library
+      that contains additional features--many of which may or may not be
+      merged into OpenResty at some point.
+   ]],
+   license = "Apache 2.0",
+   homepage = "https://github.com/Kong/lua-resty-websocket"
+}
+dependencies = {}
+build = {
+   type = "builtin",
```

Comment:
Hmmm the indentation is off across the file? It's all 3 spaces instead of 2.

Reply:
Heh, so this is the same thing as the `scm` version placeholder. The 2 or 3 rockspec files I checked at random in the Kong org are indented this way, and with [the luarocks codebase famously using 3 spaces for indentation](https://github.com/luarocks/luarocks/blob/21c84dad3ff4e0de10b813260b6a5247771d6754/.editorconfig#L11), I thought it might actually be a weird, intentional thing. I'll re-align it.